updateFeatureFlag/main.go

```diff
@@ -27,6 +27,8 @@ func init() {
 func updateFeatureFlag(flagId string, updateFeatureFlagRequest utils.UpdateFeatureFlagRequest) (events.APIGatewayProxyResponse, error) {
```

Comment:
ConditionExpression: aws.String("attribute_exists(Id)"),Here in this condition expression will evaluate to true if the Id attribute exists, even if the item does not exist. This means that the UpdateItem operation will succeed even if the feature flag with the given flagId does not exist.

Reply:
if the item doesn't exists we will get `ConditionalCheckFailedException: The conditional request failed` 
@iamitprakash 

Ref : https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Expressions.ConditionExpressions.html#Expressions.ConditionExpressions.SimpleComparisons